### PR TITLE
Add the Owner Name to differentiate when merging

### PR DESCRIPTION
**Pull request by gdutyifei**
add the Owner Name to differentiate when merging

### CODE REVIEW

**Review by appleboy — DISMISSED** [review]

**Comment by lafriks** [comment]
@appleboy fixed

File: routers/repo/pull.go [hunk 1]
```diff
@@ -615,8 +615,10 @@ func ParseCompareInfo(ctx *context.Context) (*models.User, *models.Repository, *
 	if isSameRepo {
 		headRepo = ctx.Repo.Repository
 		headGitRepo = ctx.Repo.GitRepo
+		ctx.Data["BaseName"] = headRepo.Owner.Name
 	} else {
 		headGitRepo, err = git.OpenRepository(models.RepoPath(headUser.Name, headRepo.Name))
+		ctx.Data["BaseName"] = headRepo.BaseRepo.Owner.Name
```
Comment (jonasfranz) [inline routers/repo/pull.go]: Use `baseRepo.OwnerName` instead since `headRepo.BaseRepo.Owner` might be nil.

File: routers/repo/pull.go [hunk 2]
```diff
@@ -615,8 +615,10 @@ func ParseCompareInfo(ctx *context.Context) (*models.User, *models.Repository, *
 	if isSameRepo {
 		headRepo = ctx.Repo.Repository
 		headGitRepo = ctx.Repo.GitRepo
+		ctx.Data["BaseName"] = headRepo.Owner.Name
```
Comment (jonasfranz) [inline routers/repo/pull.go]: Use `headUser.Name`

**Review by jonasfranz — DISMISSED** [review]

File: routers/repo/pull.go (hunk 1 above)
Comment (techknowlogick) [reply routers/repo/pull.go]: updated

File: routers/repo/pull.go (hunk 2 above)
Comment (techknowlogick) [reply routers/repo/pull.go]: updated

**Comment by techknowlogick** [comment]
@appleboy @JonasFranzDEV please review

**Comment by techknowlogick** [comment]
@appleboy @jonasfranz please review

**Comment by techknowlogick** [comment]
@appleboy @jonasfranz I am going to clear your reviews so it won't block other maintainers review.

**Review by techknowlogick — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**